src/components/input/styles.module.css

```diff
@@ -0,0 +1,26 @@
+.container{
+    width: 70vw;
+    height: 20vh;
+    margin:1.5rem auto;
+    background-color: var(--card-bg);
+    box-shadow: var(--card-shadow-hover);
+    transition: box-shadow 0.2s ease-in-out, tranform 0.2s ease-in-out, background-color 0.5s ease-in-out;
```

Comment:
why is background-color's timing different?

Reply:
done!